packages/www/docs/api-reference/session/get-session.mdx

```diff
@@ -9,11 +9,31 @@ metaDescription: Livepeer.com API Reference | GET retrieve a session
 
 You can retrieve a session with its ID.
 
+#### curl
+
 ```bash
 curl -H 'authorization: Bearer {api-key}' \
 "https://livepeer.com/api/session/{id}"
 ```
 
+#### JavaScript + axios
+
+```javascript
+const axios = require("axios");
```

Comment:
i'd rather suggest the inbuilt `fetch` over axios for requests; since `fetch` is built into the node ecosystem, it reduces dependencies for developers as well.